.github/CODEOWNERS

```diff
@@ -112,7 +112,7 @@
 /sdk/analysisservices/Microsoft.Azure.Management.AnalysisServices/ @athipp @taiwu @minghan
 
 # ServiceLabel: %API Management %Service Attention
-/sdk/apimanagement/Microsoft.Azure.Management.ApiManagement/       @miaojiang
+/sdk/apimanagement/Microsoft.Azure.Management.ApiManagement/       @miaojiang @melina5656
```

Comment:
revert this change since we SDK team will not be the code owner of this specific RP.

Reply:
Resolved